res/openvpn-gui-res-it.rc

```diff
@@ -172,50 +172,50 @@ END
 /* Advanced Dialog */
 ID_DLG_ADVANCED DIALOGEX 6, 18, 252, ADVANCED_DIALOG_HEIGHT
 STYLE WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU | DS_CENTER
-CAPTION "Avanzate"
+CAPTION "Opzioni avanzate"
 FONT 8, "Microsoft Sans Serif"
 LANGUAGE LANG_ITALIAN, SUBLANG_DEFAULT
 BEGIN
-    GROUPBOX "File di configurazione", 201, 6, 12, 235, 45
+    GROUPBOX "File configurazione", 201, 6, 12, 235, 45
     LTEXT "&Cartella:", ID_TXT_FOLDER, 17, 25, 32, 10
     EDITTEXT ID_EDT_CONFIG_DIR, 53, 23, 150, 12, ES_AUTOHSCROLL
     LTEXT "&Estensione:", ID_TXT_EXTENSION, 17, 40, 52, 10
     EDITTEXT ID_EDT_CONFIG_EXT, 53, 38, 25, 12, ES_AUTOHSCROLL
     PUSHBUTTON "…", ID_BTN_CONFIG_DIR, 208, 23, 25, 12
 
-    GROUPBOX "File di log", 202, 6, 62, 235, 30
+    GROUPBOX "File registro", 202, 6, 62, 235, 30
```

Comment:
"log" is accepted in Italian: https://it.wikipedia.org/wiki/Log
Not sure why you want to translate it...The meaning is pretty clear on computers, while "registro" or "registro eventi" sounds more like a forced translation, also ambiguous.

Reply:
@ordex 

"Log" accepted dind't mean suggested.
And shouldn't be the first choice for localization.
It's a typical way to not trarnslate because is known in english.
For "directory" there is an italian translation that is "cartella".
For "log" there is an italian translation that is "registro".
For "performance" there is an italian translation that is "prestazioni".
And you can see on italian TV that many person to appear more "professional" use english term instead italian terms".
Because for many users english word sounds better (but is not true if you have a italian translation for that - we are italian not english).

Reply:
You could fix wikipedia first..